apps/authentication-portal/src/main/webapp/basicauth.jsp

```diff
@@ -54,12 +54,27 @@
                     console.warn("Prevented a possible double submit event");
                 } else {
                     e.preventDefault();
+
+                    var tenantName = getParameterByName("tenantDomain");
                     var userName = document.getElementById("username");
-                    userName.value = userName.value.trim();
-                    if(userName.value){
+
+                    var usernameUserInput = document.getElementById("usernameUserInput");
+
+                    if (usernameUserInput) {
+                        var usernameUserInputValue = usernameUserInput.value.trim();
+
+                        if (getParameterByName("isSaaSApp") === "false") {
```

Comment:
Can we check whether the username already ends with the tenant domain skip the tenant appending part...to support the old behavior

Reply:
Done